wdl/transforms/biscayne/src/main/java/wdl/biscayne/parser/WdlParser.java

```diff
@@ -516,72 +516,72 @@ public String missingTerminator(String method, TerminalIdentifier terminator, Te
         public String string();
     }
     public enum WdlTerminalIdentifier implements TerminalIdentifier {
-        TERMINAL_AFTER(31, "after"),
-        TERMINAL_ALIAS(1, "alias"),
-        TERMINAL_AS(52, "as"),
-        TERMINAL_ASTERISK(61, "asterisk"),
-        TERMINAL_BOOLEAN(13, "boolean"),
-        TERMINAL_CALL(36, "call"),
-        TERMINAL_CMD_ATTR_HINT(4, "cmd_attr_hint"),
-        TERMINAL_CMD_PART(25, "cmd_part"),
-        TERMINAL_COLON(7, "colon"),
-        TERMINAL_COMMA(22, "comma"),
-        TERMINAL_DASH(29, "dash"),
-        TERMINAL_DOT(60, "dot"),
-        TERMINAL_DOUBLE_AMPERSAND(45, "double_ampersand"),
-        TERMINAL_DOUBLE_EQUAL(50, "double_equal"),
-        TERMINAL_DOUBLE_PIPE(0, "double_pipe"),
-        TERMINAL_E(27, "e"),
-        TERMINAL_ELSE(30, "else"),
-        TERMINAL_EQUAL(35, "equal"),
-        TERMINAL_ESCAPE(55, "escape"),
-        TERMINAL_EXPRESSION_PLACEHOLDER_END(3, "expression_placeholder_end"),
-        TERMINAL_EXPRESSION_PLACEHOLDER_START(6, "expression_placeholder_start"),
+        TERMINAL_AFTER(52, "after"),
```

Comment:
Out of curiosity, what do these numbers represent?

Reply:
They in some way control the internal control flow of the parser; they are unstable, possibly because there is a map data structure involved, and generating multiple parsers from the same grammar will see the numbers shift every time.

Reply:
I see. Thank you!

Reply:
Yeah, I'd avoid reading too much of anything into "why is `WdlParser.java` like this?"